lib/routes/about.js

```diff
@@ -17,17 +23,15 @@ module.exports = async (params, service) => ({
         id: service.config.name || service.packageJson.name,
         name: service.config.name || service.packageJson.name,
         description: service.config.description || service.packageJson.description,
-        version: service.packageJson.version || constants.DEFAULT_SERVICE_VERSION,
-        host: service.config.host || getServiceUrl(params),
-        protocol: service.config.protocol || params[constants.HEADER_X_FORWARDED_PROTO] || constants.DEFAULT_PROTOCOL,
+        version: service.config.version || service.packageJson.version || constants.DEFAULT_SERVICE_VERSION,
+        host: service.config.host || os.hostname(),
```

Comment:
I'm curious about the use case the `service.config.host`. Could it not always be `os.hostname()` or do you use the first for testing?